Jenkinsfile

```diff
@@ -0,0 +1,132 @@
+#!groovy
+
+import groovy.json.JsonOutput
+
+@Library('sec_ci_libs@v2-latest') _
+
+ansiColor('xterm') {
+
+  node {
+    properties([
+      parameters([
+        string(name: "SLACK_CREDENTIAL_ID", defaultValue: "25fe61e8-597e-430f-94bf-e58df726f9eb"),
+        string(name: "SLACK_CHANNEL", defaultValue: "#dcos-storage"),
+        string(name: "ALERTS_FOR_BRANCHES", defaultValue: "master")
+      ])
+    ])
+
+    stage("Verify author") {
+      def alerts_for_branches = params.ALERTS_FOR_BRANCHES.tokenize(",") as String[]
+      user_is_authorized(alerts_for_branches, params.SLACK_CREDENTIAL_ID, params.SLACK_CHANNEL)
+    }
+  }
+
+  node("mesos-ubuntu") {
+    properties([
+      parameters([
+        string(name: "GITHUB_CREDENTIAL_ID", defaultValue: "d146870f-03b0-4f6a-ab70-1d09757a51fc"),
+        string(name: "S3_REGION", defaultValue: "us-east-1"),
+        string(name: "S3_BUCKET", defaultValue: "dcos-storage-artifacts"),
+        string(name: "AWS_PROFILE_NAME", defaultValue: "dcos-storage-artifacts"),
+      ])
+    ])
+
+    checkout scm
+
+    stage("Prepare") {
+      sh("make rebuild-dev-image")
+    }
+
+    stage("Build and Test") {
+      sh("DOCKER=yes make check")
```

Comment:
maybe we could clean this up using `withEnv` to avoid expressing `DOCKER=yes` multiple times?